docs/tutorial.rst

```diff
@@ -55,18 +42,23 @@ macOS
 
 .. code-block:: console
 
-    $ pyenv install 3.7.0
-    $ pyenv virtualenv 3.7.0 cci
-    $ # Copy the following line to ~/.bash_profile to automatically activate the virtual environment in all new shells.
-    $ pyenv activate cci
+    $ python3 -m venv cci
```

Comment:
hmm, have we tested with system python3? I know there were OpenSSL issues with system python2. we may still want them to brew in their python.

Reply:
oh, good question. I'll experiment a bit

Reply:
Seems to work fine with the Python installer from python.org